testsuite/integration/basic/src/test/java/org/jboss/as/test/integration/jaxrs/cfg/ResteasyAttributeTestCase.java

```diff
@@ -153,24 +155,41 @@ public static void afterClass() throws IOException {
     //////////////////////////////////////////////////////////////////////////////////////
     /*
      * Set attributes to testable values.
+     *
+     * Note that StringTextStar is added to 'resteasy.providers" to compensate for the fact that
+     * "resteasy.use.builtin.providers" is set to "false".
      */
     static void setAttributeValues() throws IOException {
         setAttributeValue(JaxrsAttribute.JAXRS_2_0_REQUEST_MATCHING, changeBoolean(JaxrsAttribute.JAXRS_2_0_REQUEST_MATCHING.getDefaultValue()));
         setAttributeValue(JaxrsAttribute.RESTEASY_ADD_CHARSET, changeBoolean(JaxrsAttribute.RESTEASY_ADD_CHARSET.getDefaultValue()));
         setAttributeValue(JaxrsAttribute.RESTEASY_BUFFER_EXCEPTION_ENTITY, changeBoolean(JaxrsAttribute.RESTEASY_BUFFER_EXCEPTION_ENTITY.getDefaultValue()));
         setAttributeValue(JaxrsAttribute.RESTEASY_DISABLE_HTML_SANITIZER, changeBoolean(JaxrsAttribute.RESTEASY_DISABLE_HTML_SANITIZER.getDefaultValue()));
-        setAttributeValue(JaxrsAttribute.RESTEASY_DISABLE_PROVIDERS, new ModelNode(Provider1.class.getName() + "," + Provider2.class.getName()));
+        ModelNode list = new ModelNode().setEmptyList();
+        list.add(FileProvider.class.getName());
```

Comment:
Here and in other attributes that support expressions, you should do something like:
```
ModelNode value = new ModelNode(new ValueExpression("${rest.test.dummy.property:" + FileProvider.class.getName() + "}"));
list.add(value);
```
This ValueExpression stuff seems quite new, the old way which is deprecated but still heavily used is
```
ModelNode value = new ModelNode();
value.setExpression("${rest.test.dummy.property:" + FileProvider.class.getName() + "}")
```

Reply:
Done. I put expressions in one list and one map.

Reply:
I think you should use expressions everywhere where they are supported. It helps 

- make sure that they actually work in the model (although the subsystem test is good enough for that)
- make sure that they are translated/resolved properly for the runtime (not really possible in the subsystem tests)

Reply:
Ok

Reply:
Done